test/controllers/desks_controller_test.rb

```diff
@@ -46,7 +46,10 @@ class DesksControllerTest < ActionDispatch::IntegrationTest
   test "#update updates desk and responds with updated desk" do
     put desk_path(@desk, format: :json), params: { desk: { occupied: !@desk.occupied } }
 
-    assert_broadcast_on('desks', id: @desk.id, name: @desk.name, occupied: !@desk.occupied)
+    assert_broadcast_on('desks', desk_info: {id: @desk.id, name: @desk.name, occupied: !@desk.occupied},
+                        element_style: 'success',
+                        element_text: 'Not Occupied',
+                        element_id: "desk_state_#{@desk.id}")
```

Comment:
@DominikSchmidtlein @erickadbay There must be someway to use the helpers in this test class? I couldn't figure out how to do it without creating static methods in DeskHelper

Reply:
There is. I believe if you include the module at the top `include DesksHelper` you can use the methods. However, I still believe this logic belongs more in javascript-land than in controller land.

Reply:
Waiting for action on this.